### DISCUSSION TITLE
How do I setup Codec manually?

### DISCUSSION THREAD

**Discussion by forceassaulter** [PyAV-Org/PyAV#930] [1. Help]
I need to stream the video in mpeg format from buffer, hence the buffer is too short for FFMPEG to detect codec automatically, how do I setup codec manually?

The error:
`probed stream 0 failed
start time for stream 0 is not set in estimate_timings_from_pts
Could not find codec parameters for stream 0 (Video: none, none): unknown codec
Consider increasing the value for the 'analyzeduration' and 'probesize' options`

**Comment by kkangshen** [comment]
I encountered the same problem when using pyav parsing, and I don't know how to solve it
